source/sync/app-builder/tiered-device-sync/index.txt

```diff
@@ -179,6 +179,11 @@ Set up and run the Tiered Sync server on the host.
            - The public DNS of your host. This should resemble something
              like ``ec2-13-58-70-88.us-east-2.compute.amazonaws.com``.
 
+      .. note::
```

Comment:
We try to avoid stacking directives, so I think we could just add this information to the `hostname` value above. Maybe append it after ln 180 - something like:

> like ``ec2-13-58-70-88.us-east-2.compute.amazonaws.com``.
> For localhost, set this value to `localhost:80`.